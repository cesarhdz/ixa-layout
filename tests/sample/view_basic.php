<h1><?php echo ucwords('post title') ?></h1>
<p>Post Content</p>