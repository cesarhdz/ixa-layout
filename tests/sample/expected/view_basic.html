<h1>Post Title</h1>
<p>Post Content</p>